big_tests/tests/graphql_last_SUITE.erl

```diff
@@ -0,0 +1,401 @@
+-module(graphql_last_SUITE).
+
+-compile([export_all, nowarn_export_all]).
+
+-import(distributed_helper, [mim/0, require_rpc_nodes/1, rpc/4]).
+-import(graphql_helper, [execute_user/3, execute_auth/2, user_to_bin/1, user_to_jid/1,
+                         get_ok_value/2, get_err_msg/1, get_err_code/1]).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-define(assertErrMsg(Res, ContainsPart), assert_err_msg(ContainsPart, Res)).
+-define(assertErrCode(Res, Code), assert_err_code(Code, Res)).
+
+-define(NONEXISTENT_JID, <<"user@user.com">>).
+-define(DEFAULT_DT, <<"2022-04-17T12:58:30.000000Z">>).
+
+suite() ->
+    require_rpc_nodes([mim]) ++ escalus:suite().
+
+all() ->
+    [{group, user_last},
+     {group, admin_last},
+     {group, admin_last_old_users}].
+
+groups() ->
+    [{user_last, [parallel], user_last_handler()},
+     {admin_last, [parallel], admin_last_handler()},
+     {admin_last_old_users, [], admin_old_users_handler()}].
+
+user_last_handler() ->
+    [user_set_last,
+     user_get_last,
+     user_get_other_user_last].
+
+admin_last_handler() ->
+    [admin_set_last,
+     admin_try_set_nonexistent_user_last,
+     admin_get_last,
+     admin_get_nonexistent_user_last,
+     admin_try_get_nonexistent_last,
+     admin_count_active_users,
+     admin_try_count_nonexistent_domain_active_users].
+
+admin_old_users_handler() ->
+    [admin_list_old_users_domain,
+     admin_try_list_old_users_nonexistent_domain,
+     admin_list_old_users_global,
+     admin_remove_old_users_domain,
+     admin_try_remove_old_users_nonexistent_domain,
+     admin_remove_old_users_global,
+     admin_user_without_last_info_is_old_user,
+     admin_logged_user_is_not_old_user].
+
+init_per_suite(Config) ->
+    HostType = domain_helper:host_type(),
+    SecHostType = domain_helper:secondary_host_type(),
+    Config1 = escalus:init_per_suite(Config),
+    Config2 = dynamic_modules:save_modules(HostType, Config1),
```

Comment:
oh, there is `save_modules(Node = #{node := NodeName}, HostTypes, Config)` for multiple host types :)